timesketch/lib/analyzers/base_sessionizer_test.py

```diff
@@ -27,100 +25,96 @@ def test_analyzer(self):
                 MockDataStore)
     def test_same_session(self):
         """Test multiple events in the same session are allocated correctly."""
-        with mock.patch.object(
```

Comment:
Mocking the event_stream function is no longer needed. The ElasticsearchDataStore bahaviour inside the function is mocked.